recipe/meta.yaml

```
@@ -1,14 +1,15 @@
 {% set name = "openquake.engine" %}
-{% set version = "3.19.0" %}
+{% set version = "3.21.0" %}
+{% set python_min = "3.8" %}
```

I changed to 3.8 b/c that is the lowest one where the min numba required here is resolved. However, it seems that openquake.engine requires Python 3.9 as a minimum. See https://github.com/gem/oq-engine/blob/b75b934f1b3400f1bd77c3488c817220e73fd3fd/setup.py#L24

```suggestion
{% set python_min = "3.9" %}
```